strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -187,6 +194,21 @@ public Validator localValidatorFactoryBean()
     {
         return new LocalValidatorFactoryBean();
     }
+    
+    @Bean("loggingMgmtDirectoryListingService")
+    public DirectoryListingService getLoggingManagementDirectoryListingService()
+    {
+    	String baseUrl = StringUtils.chomp(configurationManager.getConfiguration().getBaseUrl(), "/");
+
+        return new DirectoryListingServiceImpl(String.format("%s/api/logging", baseUrl));
+    }
+    
+    @Bean("browseRepoDirectoryListingService")
+    public DirectoryListingService getBrowseRepoDirectoryListingService()
+    {
+    	String baseUrl = StringUtils.chomp(configurationManager.getConfiguration().getBaseUrl(), "/");
```

Comment:
Cosmetic: Could you please follow our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks!

Also, we don't like to abbreviate the names of beans like this. Could you please rename this to `browseRepoDirectoryListingService`? :)

Reply:
@carlspring , I suppose you meant to point out `loggingMgmtDirectoryListingService` ? Because the other bean is already appropriately name. I will change this to `loggingManagementDirectoryListingService` instead.

Reply:
Actually, I believe you've abbreviated both of their names. One has `repo`, the other -- `mgmt`. If you could use their full names, that would be much better, thanks! :)

We had a similar discussion with @bono007 on one of his pull requests last week. It's a matter of convention that we follow..

Reply:
Fair enough.. in my defence, I have been using the word `repo` as a common noun for quite sometime now, that's why it didn't hit me as an abbreviation. I'll do the changes now.

Reply:
Thank you! :)